src/Keywords/CachedArrayKeywords.php

```diff
@@ -19,13 +19,15 @@
  */
 class CachedArrayKeywords extends ArrayKeywords
 {
+    private const I18N_FILE_LOCATION = __DIR__ . '/../../i18n.php';
+
     /**
      * Initializes holder with file.
      *
-     * @param string $file Cached array path
+     * @param string|null $file Cached array path
      */
-    public function __construct($file)
+    public function __construct($file = null)
     {
-        parent::__construct(include $file);
+        parent::__construct(include $file ?? self::I18N_FILE_LOCATION);
```

Comment:
should we change this to `require` while we're here? include would still fail if the file didn't exist, as the `false` wouldn't match the `array` typehint on the parent constructor - but `require` would give a better error message?

Reply:
Yes, that makes sense, updated